src/gui/TextEditor.h

```diff
@@ -60,6 +62,9 @@ class TextEditor {
     UndoAction* setColor(Color color);
 
 private:
+    void setTextToPangoLayout(PangoLayout* pl) const;
```

Comment:
It's not obvious to me what this does, so I suggest adding a brief doc comment explaining its purpose.